src/app/Http/Controllers/ConnectController.php

```diff
@@ -35,21 +35,35 @@ public function __construct(UserRepository $repository, AuthValidatorInterface $
     public function connect()
     {
         try {
-            $user = $this->repository->findByField('username', 'admin@vmms.vn')->first();
-            
+            $token = JWTAuth::getToken();
+            if (empty($token)) {
+                return response([
```

Comment:
@minhquang19 Chỗ này em throw exception `UnauthorizedHttpException` để đúng status code trả về là 401 nhé.
Danh sách các exception em xem thêm ở đây: https://github.com/dingo/api/wiki/Errors-And-Error-Responses